src/LetsEncrypt.Azure/McMaster.AspNetCore.LetsEncrypt.Azure.csproj

```diff
@@ -7,9 +7,10 @@
     <IsPackable>true</IsPackable>
     <PackageIcon>images/le-logo-standard.png</PackageIcon>
     <Description>Provides API for configuring ASP.NET Core to use Let's Encrypt.org to automaticaly generate HTTPS certificates and store them in Azure Key Vault.</Description>
-    <PackageDescription>$(Description)
+    <PackageDescription>
+      $(Description)
 
-See https://nuget.org/packages/McMaster.AspNetCore.LetsEncrypt for more details.
+      See https://nuget.org/packages/McMaster.AspNetCore.LetsEncrypt for more details.
     </PackageDescription>
```

Comment:
I'm guessing this was an unintentional change by the IDE. It adds extra whitespace to the description.

Reply:
I rarely try to override IDE auto changes, as it's generally futile in the end. If this is a deal breaker, let me know.